packages/headless/src/controllers/core/triggers/headless-core-notify-trigger.test.ts

```diff
@@ -0,0 +1,153 @@
+import {logNotifyTrigger} from '../../../features/triggers/trigger-analytics-actions';
+import {triggerReducer as triggers} from '../../../features/triggers/triggers-slice';
+import {
+  buildMockSearchEngine,
+  MockedSearchEngine,
+} from '../../../test/mock-engine-v2';
+import {createMockState} from '../../../test/mock-state';
+import {NotifyTrigger} from '../../core/triggers/headless-core-notify-trigger';
+import {buildCoreNotifyTrigger} from './headless-core-notify-trigger';
```

Comment:
I believe you'll have to add `.js` extensions to your relative imports, otherwise the build will fail

(same comment everywhere it applies)